tests/processes/CredentialProcess.Worker.Tests/CredentialCreationProcessTypeExecutorTests.cs

```diff
@@ -117,7 +117,7 @@ public async Task InitializeProcess_WithNotExistingProcess_ThrowsNotFoundExcepti
             .Returns(new ValueTuple<bool, Guid>(false, Guid.Empty));
 
         // Act
-        async Task Act() => await _sut.InitializeProcess(validProcessId, Enumerable.Empty<ProcessStepTypeId>()).ConfigureAwait(false);
+        async Task Act() => await _sut.InitializeProcess(validProcessId, Enumerable.Empty<ProcessStepTypeId>());
```

Comment:
async + await is redunant - please remove

Reply:
adjusted as discussed